modules/omx/module.c

```diff
@@ -0,0 +1,149 @@
+#include "omx.h"
```

Comment:
a doxygen header is needed. you can use the same style as other module.
please keep the copyright creytiv line, you can also add your own name if you wish.